training/model/Items.swift

```diff
@@ -0,0 +1,36 @@
+//
+// Created by Benedykt Ziobro on 15/11/2017.
+// Copyright (c) 2017 Benedykt Ziobro. All rights reserved.
+//
+
+import Foundation
+
+
+struct Items {
```

Comment:
odd... I would suggest to create model for single item with convenience init and initialize array of A objects after receiving it from rest